go/client/chat_cli_flag_utils.go

```diff
@@ -113,12 +121,15 @@ func getInboxFetcherActivitySortedFlags() []cli.Flag {
 
 func parseConversationTopicType(ctx *cli.Context) (topicType chat1.TopicType, err error) {
 	switch t := strings.ToLower(ctx.String("topic-type")); t {
+	case "":
```

Comment:
Is this for when user explicitly set `--topic-type=""` or something else? The default is "chat". Also I think this could be `case "", "chat":`